docs/index.adoc

```diff
@@ -4776,7 +4777,7 @@ class ExecParameterConsumer : IParameterConsumer {
 }
 ----
 
-==== `IParameterPreprocessor` Parser Plugin
+==== IParameterPreprocessor Parser Plugin
```

Comment:
This probably looks better. :-) Should we do the same for IParameterConsumer?

Reply:
That said, there are several other places in the manual where backticks are used for class names in headings.
Maybe best to keep the backticks.

Reply:
I never wanted to removed the backticks. They simply were a hindrance to use the section title as cross reference target, that's why I removed them. Fortunaltely, I could overcome this difficulty, and the backticks are now in place again.